forte/data/data_store.py

```diff
@@ -26,9 +26,15 @@ def __init__(self):
         Each `entry data` in the `entry list` is represented by a list of attributes,
         For example, annotation type entry has the following format:
         [<begin>, <end>, <tid>, <entry_type>, <attr_1>, <attr_2>, ..., <attr_n>].
-
-        The first four fields are compulsory for every `entry data`. They are always in
-        the order of `begin`, `end`, `tid` and `entry type`, .
+        Group type entry has the following format:
+        [<member_type>, <[members_tid_list]>, <tid>, <entry_type>, <attr_1>,
+            <attr_2>, ..., <attr_n>].
+        Link type entry has the following format:
+        [<parent_tid>, <child_tid>, <tid>, <entry_type>, <attr_1>, <attr_2>, ..., <attr_n>].
+
+        The first four fields are compulsory for every `entry data`.
+        For example, first four fields of annotations entries are always in
+        the order of `begin`, `end`, `tid` and `entry type` .
         `Begin` and `end` are the begin and end character index of this entry in the text.
```

Comment:
Now since you added links/groups, this statement becomes inaccurate. Not every entry has begin or end.

Reply:
Links and groups still maintain the structure that 'first four fields are compulsory'. Do you mean I should rephrase line #38?

Reply:
Right, an entry does not necessarily have entry, so line 38 is wrong.